decidim-elections/db/migrate/20210330123606_add_voting_ballot_style.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddVotingBallotStyle < ActiveRecord::Migration[5.2]
+  def change
+    create_table :decidim_votings_ballot_styles do |t|
+      t.string :title
+      t.string :code, index: true
+      t.references :decidim_votings_voting,
+                   null: false,
+                   index: { name: "decidim_votings_votings_ballot_styles" }
+
+      t.timestamps
+    end
+
+    create_join_table :decidim_votings_ballot_styles, :decidim_elections_questions, table_name: "decidim_votings_ballot_style_questions" do |t|
```

Comment:
Wow I didn't know about the `create_join_table` 😮

Reply:
✨

Reply:
me neither! 💎